addOns/graphql/graphql.gradle.kts

```diff
@@ -0,0 +1,16 @@
+version = "1"
```

Comment:
This may as well use semver and start at 0.1.0 or even 0.0.1?

Reply:
Okay. I was not aware of semver, I'll update this. Since this is not exactly a release I think we could go with 0.0.1.